service/operator/service.go

```diff
@@ -13,14 +13,16 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
 
+	"github.com/giantswarm/azure-operator/client/azure"
 	"github.com/giantswarm/azure-operator/flag"
 )
 
 // Config represents the configuration used to create an Operator service.
 type Config struct {
 	// Dependencies.
-	Logger    micrologger.Logger
-	K8sClient kubernetes.Interface
+	AzureConfig *azure.Config
```

Comment:
The Azure config is stored rather than the clients to avoid the auth tokens expiring.